### PR TITLE
"Where is my Ruby Method?" article

### PR DESCRIPTION
**Pull request by sethjeffery**
@quipper/web-devs 

Added new simple article.
Also fixed the crazy color cycle on chrome.

### CODE REVIEW

File: _posts/2015-10-12-where-is-my-ruby-method-defined.markdown [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+{% endhighlight %}
+
+
+## source_location
+
+After the initial panic wears off, the sane thing to do in this situation is to find out what `present?` is actually doing.
+Fortunately, Ruby 1.9+ allows us to find exactly where a method is defined: `source_location` !
+
+{% highlight ruby %}
+Chapter.new.methods(:present?).source_location
```
Comment (yuya-takeyama) [inline _posts/2015-10-12-where-is-my-ruby-method-defined.markdown]: Not `methods` but `method`?

Comment (sethjeffery) [inline _posts/2015-10-12-where-is-my-ruby-method-defined.markdown]: Thanks

**Comment by yuya-takeyama** [comment]
LGTM :+1:

**Comment by banyan** [comment]
LGTM :smile:

**Pull request merged**